Data-Engineer.md

```diff
@@ -0,0 +1,63 @@
+# Work Sample for Data Engineer
+
+Solve the following data pipeline problems using a DAG (Directed Acyclic Graph) oriented tool, such as Pachyderm, Airflow, Dagster, etc.
```

Comment:
I would add a bit more context around _why_ this is important for the engineer to utilize a DAG in the following problem statements, and try to incorporate it a bit more. I would ask for some artifacts related to the DAG, such as pipeline specs or diagrams to be included.